rpm/stackdriver-agent.spec

```diff
@@ -234,7 +235,7 @@ popd
 %{__make} install DESTDIR="%{buildroot}"
 
 %{__install} -Dp -m0755 %{SOURCE200} %{buildroot}/%{_initddir}/stackdriver-agent
-%{__install} -Dp -m0644 %{SOURCE201} %{buildroot}%{_sysconfdir}/collectd.conf
+%{__install} -Dp -m0644 %{SOURCE201} %{buildroot}/%{_confdir}/collectd.conf
```

Comment:
Would it make sense to "double write"? I.e., put collectd.conf in both the new and old locations? Or put it in the new location only and put a redirect in the old location that points to the new location?

Reply:
Neither of your suggested approaches is supported by collectd...

Reply:
Yeah, got it, thanks. For the fourth time. Maybe I should have asked this as a meta-comment.